src/rest-service/services/login.service.ts

```diff
@@ -0,0 +1,15 @@
+import { sign } from 'jsonwebtoken';
+import { findUserByLogin } from '../data-access/users.data-access';
+
+export const handleGetTokenService = async (
+    login: string,
+    password: string,
+) => {
+    const user = await findUserByLogin(login);
+
+    if (user && user.password === password && !user.isDeleted) {
+        return sign({ login, password }, process.env.SECRET as string);
```

Comment:
I suggest not to return the password in the token

Reply:
updated